regression/cbmc-library/export-core-goto/test-correct.desc

```diff
@@ -0,0 +1,16 @@
+CORE
+test.c
+--export-core-goto exported.core.goto
+^Parsing test.c$
+^Converting$
+^Type-checking test$
+^Generating GOTO Program$
+^Adding CPROVER library (x86_64)$
```

Comment:
This is not going to work on M1 systems (the string searched for is `arm64`.

If we need to test that just the success indicator sentence is present, why do we need the tests to be so strong (by requiring multiple regexes to match in an `and` fashion)?